### PR TITLE
bump version

### PR DESCRIPTION
**Pull request by pgajdos**

### CODE REVIEW

File: package/yast2-fonts.changes [hunk 1]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Feb 20 13:49:43 UTC 2018 - pgajdos@suse.com
+
+- bump version
```
Comment (jreidinger) [inline package/yast2-fonts.changes]: well, I think you need some reasoning and also bsc or fate to pass automatic checks

Comment (pgajdos) [reply package/yast2-fonts.changes]: @lslezak, so the question remains. How can I include yast2-fonts in openSUSE:15.0? It seems I cannot follow your workflow ;).

Comment (ancorgs) [reply package/yast2-fonts.changes]: Those automatic checks only apply to packages that are both in SLE and openSUSE. Isn't it?

Comment (jreidinger) [reply package/yast2-fonts.changes]: @ancorgs maybe in BS, but we do it everywhere. We do not have flag to recognize that it is opensuse only.

**Comment by mchf** [comment]
Deprecated by https://github.com/yast/yast-fonts/pull/31

**Pull request closed without merging**